ansible/roles/master/tasks/download_bins.yml

```diff
@@ -5,13 +5,9 @@
 - name: Create directory for current Kubernetes release
   file: path={{ kube_current_release_directory }} state=directory
 
-- name: Set a list of kube master binaries to download
-  set_fact:
-    kube_binaries: "{{ kube_master_binaries }}"
-
 - name: Extend the list of kube master binaries with kubelet
   set_fact:
-    kube_binaries: "{{ kube_master_binaries }} kubelet"
+    kube_master_binaries: "{{ kube_master_binaries + ['kubelet'] }}"
```

Comment:
I would like to keep the intermediate variable as the kubelet is not really a master binary. What about to use ``{{kube_binaries|default([])}}`` in the ``with_items``. Your solution is syntactically clean but semantically can lead to confusion.